elements/lisk-p2p/test/integration/p2p.ts

```diff
@@ -778,16 +763,18 @@ describe('Integration tests for P2P library', () => {
 
 				// For each peer of firstP2PNode, check that the firstP2PNode's P2PPeerInfo was updated with the new height.
 				for (let p2pNode of p2pNodeList.slice(1)) {
-					const networkStatus = p2pNode.getNetworkStatus();
-					const firstNodeInConnectedPeer = networkStatus.connectedPeers.find(
-						peerInfo => peerInfo.wsPort === firstP2PNode.nodeInfo.wsPort,
-					);
+					const firstNodeInConnectedPeer = p2pNode
+						.getConnectedPeers()
+						.find(peerInfo => peerInfo.wsPort === firstP2PNode.nodeInfo.wsPort);
+
+					const newPeers = p2pNode['_peerBook'].newPeers;
+					const triedPeers = p2pNode['_peerBook'].triedPeers;
 
-					const firstNodeInNewPeer = networkStatus.newPeers.find(
+					const firstNodeInNewPeer = newPeers.find(
 						peerInfo => peerInfo.wsPort === firstP2PNode.nodeInfo.wsPort,
 					);
 
-					const firstNodeInTriedPeer = networkStatus.triedPeers.find(
+					const firstNodeInTriedPeer = triedPeers.find(
 						peerInfo => peerInfo.wsPort === firstP2PNode.nodeInfo.wsPort,
 					);
 
```

Comment:
Like this test can be easily modified to expect the height change in `getAllPeers`